### ISSUE TITLE
Add padding to the home page

### ISSUE THREAD

**Issue by RushilJalal** [0xAquaWolf/portfolio#9]
There is no space between the edge and the text on the left side.
I will add padding to make the design look better.
I would like to work on this issue. Thanks.

![image](https://github.com/user-attachments/assets/a41225ce-a9b1-4285-87ab-958a621b1d3f)

**Comment by 0xAquaWolf** [comment]
Yeah i noticed on smaller screens that it looks like this i'm open to a pull request if you want, you can submit one and i'll review it.

**Comment by RushilJalal** [comment]
What screen size are you referring to when u said 'smaller screens', because the screenshot I provided is from a laptop.
Sure, I will open a PR to fix this issue soon.

**Comment by 0xAquaWolf** [comment]
I mostly code on a 16 inch macbook pro but i noticed it on a 13 inch macbook air that it didn't have padding and i had it on my todo's but never got around to it, just saw your PR will approve it now, i had to fix a few bugs first because the build was failing but its working now
